kubetest2-tf/deployer/deployer.go

```diff
@@ -81,6 +82,7 @@ type deployer struct {
 	Playbook              string            `desc:"name of ansible playbook to be run"`
 	ExtraVars             map[string]string `desc:"Passes extra-vars to ansible playbook, enter a string of key=value pairs"`
 	SetKubeconfig         bool              `desc:"Flag to set kubeconfig"`
+	TargetProvider		  string			`flag:"~target-provider" desc:"provider value to be used(powervs, vpc)"`
```

Comment:
I think flag tag is not required, can you check ones?

Reply:
and also for all the go files you have added

Reply:
Removed the flag tag.